pnpm-lock.yaml

```diff
@@ -19982,7 +19989,7 @@ packages:
       '@apideck/better-ajv-errors': 0.3.6_ajv@8.11.0
       '@babel/core': 7.17.10
       '@babel/preset-env': 7.18.6_@babel+core@7.17.10
-      '@babel/runtime': 7.18.6
+      '@babel/runtime': 7.19.0
```

Comment:
From https://github.com/Seneca-CDOT/telescope/issues/3680, interesting to see that this is listed as a `transitivePeerDependency`.  We should add it as a dep, I guess?